src/components/InspirationSection.vue

```diff
@@ -80,20 +55,78 @@
 
 <script>
 import TheButton from './shared/TheButton.vue'
+import TheWorkout from './shared/TheWorkout.vue'
 import TheExercise from './shared/TheExercise.vue'
+import { supabase } from '../supabase'
 
 export default {
   name: 'InspirationSection',
   components: {
     TheButton,
+    TheWorkout,
     TheExercise
-  }
+  },
+  data() {
+    return {
+      workouts: [],
+      isError: false,
+      workoutNumber: 0
+    }
+  },
+  computed: {
+    message() {
+      return this.isError ? 'Ooops, something went wrong when fetching data. Try refreshing the page.' : 'Take a look at other’s workouts below!'
+    },
+    currentWorkout() {
+      return this.workouts.filter((workout) => workout === this.workouts[this.workoutNumber])
+    }
+  },
+  methods: {
+    increaseWorkoutNumber() {
+      this.workoutNumber += 1
+
+      if (!this.workouts[this.workoutNumber]) {
+        this.workoutNumber = 0
+      }
+    },
+    decreaseWorkoutNumber() {
+      this.workoutNumber -= 1
+
+      if (!this.workouts[this.workoutNumber]) {
+        this.workoutNumber = this.workouts.length - 1
+      }
+    }
+  },
+  async mounted() {
+    const fetchWorkouts = async () => {
+      const response = await supabase.from('Workouts').select()
+      if (response.error === null) {
+        const { data } = response
+
+        // eslint-disable-next-line max-len
+        this.workouts = data.map(({ workout: { workout_name, exercises } }) => ({ workout_name, exercises }))
+
```

Comment:
Do not omit max-len errors in scripts.

Reply:
```js
        this.workouts = data.map(
          // eslint-disable-next-line camelcase
          ({ workout: { workout_name, exercises } }) => ({ exercises, name: workout_name })
        )
```

Reply:
Notice that I've changed `workout_name` to `name`. You should keep camelCase  naming when possible, beside that a `workout` object should have a `name`, not a `workout_name`

Reply:
**All corrected**

I didn't want to name the workout name as "name" because I was afraid of "name" being invalid variable name, like it happened once when I used "key" as a variable and something broke. Also, forgot I can change names when destructuring 😛 And I understand I didn't give it enough thougth when I was coming up with the names in supabase since I didn't realize I would be actually using them in my code 😅